.github/CODEOWNERS

```diff
@@ -33,6 +33,7 @@
 /avm/res/app/managed-environment/ @Azure/avm-res-app-managedenvironment-module-owners-bicep @Azure/avm-module-reviewers-bicep
 /avm/res/app-configuration/configuration-store/ @Azure/avm-res-appconfiguration-configurationstore-module-owners-bicep @Azure/avm-module-reviewers-bicep
 /avm/res/automation/automation-account/ @Azure/avm-res-automation-automationaccount-module-owners-bicep @Azure/avm-module-reviewers-bicep
+/avm/res/azure-stack-hci/cluster/ @Azure/avm-res-azurestackhci-cluster-module-owners-bicep @Azure/avm-module-reviewers-bicep
```

Comment:
The team may not exist or is not placed in the hierachy as expected

Reply:
Not sure what I missed here:
![image](https://github.com/user-attachments/assets/c34301ee-59e5-4d78-a2c4-bae872081daa)

Reply:
Hey @mbrat2005, there's a chance that @matebarabas did not yet approve the parent-child relationship :) I'll doublecheck the values regardless.

Reply:
Looks like we're good now. Resolving :)

Reply:
@mbrat2005, yes, the "owners" group is set up correctly, but you don't seem to have the "contributors" group just yet. Can you please make sure you create that too? `avm-res-azurestackhci-cluster-module-contributors-bicep` --> this needs to be added to `avm-module-contributors-bicep`. Thanks!

Reply:
Yes, created!